contributing/dataset-ingestion/example-template/example-geoglam-ingest.ipynb

```diff
@@ -0,0 +1,502 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "id": "1b78106c",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: Example Ingestion Workflow for Uploading Data to the VEDA Dashboard\n",
+    "description: A walk through of the ingestion workflow for data providers who want to add a new dataset to the VEDA Dashboard.\n",
+    "author: Jonas Sølvsteen, Kathryn Berger\n",
+    "date: July 25, 2023\n",
+    "updated: August 15, 2024\n",
+    "execute:\n",
+    "  freeze: true\n",
+    "  cache: true\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a34abb82",
+   "metadata": {},
+   "source": [
+    "# Approach"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d866531f-d8c5-45bc-ad55-51b1a385beac",
+   "metadata": {},
+   "source": [
+    "This notebook is intented to be used as a reference for data providers who want to add new datasets to the VEDA Dashboard. As always it is important that the data provider has read the documentation for [Data Ingestion](https://nasa-impact.github.io/veda-docs/contributing/dataset-ingestion/) before moving forward with this notebook example. \n",
+    "\n",
+    "For example purposes, we will walk the end user through adding the GEOGLAM June 2023 dataset directly to the VEDA Dashboard. \n",
+    "\n",
+    "1. Validate the GeoTIFF\n",
+    "2. Upload the file to the staging S3 bucket (`veda-data-store-staging`)\n",
+    "3. Use the workflows-api ([staging.openveda.cloud/api/workflows/docs](https://staging.openveda.cloud/api/workflows/docs)) to generate STAC metadata for the file and add to the staging STAC catalog ([staging.openveda.cloud](https://staging.openveda.cloud))\n",
+    "\n",
+    "When the data has been published to the STAC metadata catalog for this `geoglam` collecition, which is already configured for the dashboard, it will be available in the [VEDA Dashboard](https://www.earthdata.nasa.gov/dashboard)"
```

Comment:
```suggestion
    "When the data has been published to the STAC metadata catalog for this `geoglam` collection, which is already configured for the dashboard, it will be available in the [VEDA Dashboard](https://www.earthdata.nasa.gov/dashboard)"
```